scripts/build/TestImpactAnalysis/tiaf.py

```diff
@@ -18,16 +18,227 @@
 
 logger = get_logger(__file__)
 
+
 class TestImpact:
-    def __init__(self, config_file: str):
+
+    
+
```

Comment:
uber nit: Are all these line breaks necessary (I think python recommends two between statements/functions is that correct?

Reply:
Nice one thanks!